core/client/app/services/config.js

```diff
@@ -2,27 +2,24 @@ import Ember from 'ember';
 
 const {Service, _ProxyMixin, computed} = Ember;
 
-function isNumeric(num) {
+function _isNumeric(num) {
```

Comment:
I'm :+1: on it, it's following the same semantics as the other `isBlank`, `isEmpty`, etc helper methods and makes it easily searchable if it's ever needed elsewhere.

@acburdine are you able to change the name from `_isNumeric` to `isNumeric` please? The convention for the `_` prefix is to indicate that a method or instance variable should be considered private, helper functions don't really fall into that category.